files/chroot_build.sh

```diff
@@ -75,18 +76,25 @@ echo >&2 "===]> Info: Install window manager... "
 
 apt-get install -y -qq -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" \
   plymouth-theme-spinner \
+  plymouth-theme-kubuntu-logo \
+  plymouth-theme-kubuntu-text \
   plymouth-theme-ubuntu-text \
-  ubuntu-desktop-minimal \
-  ubuntu-gnome-wallpapers \
-  snapd
+  kde-plasma-desktop \
+  kubuntu-settings-desktop \
+  plasma-discover \
+  kubuntu-wallpapers \
+  kde-plasma-desktop \
+  sddm-theme-breeze \
+  snapd \
+  firefox
```

Comment:
You cannot install snap packages in chroot, so this shall fail to install. So probably remove firefox.

Reply:
ok